R/uvhydrograph-render.R

```diff
@@ -576,6 +602,10 @@ getPrimaryPlotConfig <- function(timeseries, name, label,
           lines = append(list(x=x, y=y, side=2, ylim=ylim, ylab=label, legend.name=paste(styles[['corr_UV_lbl']], label)), styles[['corr_UV_lines']]),
           view = list(side=2, log=doLog)
           ),
+      estimatedPrimary = list(
+          lines = append(list(x=x, y=y, side=2, ylim=ylim, ylab=label, legend.name=paste(styles[['est_UV_lbl']], label)), styles[['est_UV_lines']]),
+          view = list(side=2, log=doLog)
+          ),
```

Comment:
This is so `ylab=label` can be added for estimated data.  One of the side effects of not having non-estimated data to plot is that there is no y-axis label on the plot.  More elegant solutions are welcome!